### PR TITLE
add new parser - Mobsfscan

### PR DESCRIPTION
**Pull request by ansidorov**
mobsfscan is a static analysis tool that can find insecure code patterns in your Android and iOS source code. Supports Java, Kotlin, Swift, and Objective C Code. 

It could be found at: https://github.com/MobSF/mobsfscan

### CODE REVIEW

**Review by Maffooch — DISMISSED** [review]
Is this different from 

https://github.com/DefectDojo/django-DefectDojo/tree/master/dojo/tools/mobsf

**Comment by ansidorov** [comment]
> Is this different from
> 
> https://github.com/DefectDojo/django-DefectDojo/tree/master/dojo/tools/mobsf

Yes, mobsf (https://github.com/MobSF/Mobile-Security-Framework-MobSF) more complex tool with different features.

**Review by Maffooch — COMMENTED** [review]
@damiencarol this one looks good to me, do you agree?

**Comment by damiencarol** [comment]
@Maffooch wait, I want to confirm that it's the new scanner of MOBSF

**Review by damiencarol — CHANGES_REQUESTED** [review]
Want to take time to review this one

File: dojo/unittests/tools/test_mobsfscan_parser.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            self.assertEqual("app/src/main/java/com/routes/domain/analytics/event/Signatures.kt", finding.file_path)
+            self.assertEqual(10, finding.line)
+
+        with self.subTest(i=2):
+            finding = findings[2]
+            self.assertEqual("android_prevent_screenshot", finding.title)
+            self.assertEqual("Low", finding.severity)
+            self.assertEqual(1, finding.nb_occurences)
+            self.assertIsNotNone(finding.description)
+            self.assertEqual('200', finding.cwe)
```
Comment (damiencarol) [inline dojo/unittests/tools/test_mobsfscan_parser.py:48]:
```suggestion
            self.assertEqual(200, finding.cwe)
```
cwe attribute is an integer in the current model.

**Review by damiencarol — CHANGES_REQUESTED** [review]

**Comment by damiencarol** [comment]
@ansidorov found a minor thing.

**Comment by ansidorov** [comment]
> @ansidorov found a minor thing.

Fixed

**Review by damiencarol — APPROVED** [review]

**Comment by damiencarol** [comment]
> 
> 
> @damiencarol this one looks good to me, do you agree?

Yes, made a minor comment but it's ok for me.

**Review by Maffooch — APPROVED** [review]

**Pull request merged**